src/native/RnExecutorchModules.ts

```diff
@@ -72,44 +72,44 @@ const StyleTransfer = StyleTransferSpec
     );
 
 class _ObjectDetectionModule {
```

Comment:
Let's propagate return types from turbo modules specs like so:
```suggestion
class _ObjectDetectionModule {
  static async forward(input: string) : ReturnType<ObjectDetectionInterface["forward"]> {
```

Reply:
`import { Spec as ObjectDetectionInterface } from './NativeObjectDetection';`

Reply:
Do this for all methods for other classes in this file too.